src/Solver/time_stepping/TimeCluster.cpp

```diff
@@ -140,7 +142,8 @@ seissol::time_stepping::TimeCluster::TimeCluster(unsigned int i_clusterId, unsig
     m_globalClusterId(i_globalClusterId),
     m_profilingId(profilingId),
     dynamicRuptureScheduler(dynamicRuptureScheduler),
-    seissolInstance(seissolInstance)
+    seissolInstance(seissolInstance),
+    m_dynamicRuptureKernel(damageparameters)
```

Comment:
No need to pass `damageparameters`, since you can access it through `seissolInstance.getSeisSolParameters()....`